PhysicsTools/NanoAOD/python/leptonTimeLifeInfo_common_cff.py

```diff
@@ -48,12 +48,12 @@
 # secondary vertex
 svVars = cms.PSet(
     # SV
-    hasRefitSV = Var("hasSV()", bool, doc="has SV refit using miniAOD quantities"),
-    refitSVx = Var("?hasSV()?sv().x():0", float, doc="x coordinate of SV", precision=10),
-    refitSVy = Var("?hasSV()?sv().y():0", float, doc="y coordinate of SV", precision=10),
-    refitSVz = Var("?hasSV()?sv().z():0", float, doc="z coordinate of SV", precision=10),
-    refitSVchi2 = Var("?hasSV()?sv().normalizedChi2():0", float, doc="reduced chi2, i.e. chi2/ndof, of SV fit", precision=8),
-    #refitSVndof = Var("?hasSV()?sv().ndof():0", float, doc="ndof of SV fit", precision=8),#MB: not important
+    hasRefitSV = Var("hasSV()", bool, doc="has SV refit using miniAOD quantities", lazyEval=True),
+    refitSVx = Var("?hasSV()?sv().x():0", float, doc="x coordinate of SV", precision=10, lazyEval=True),
+    refitSVy = Var("?hasSV()?sv().y():0", float, doc="y coordinate of SV", precision=10, lazyEval=True),
+    refitSVz = Var("?hasSV()?sv().z():0", float, doc="z coordinate of SV", precision=10, lazyEval=True),
+    refitSVchi2 = Var("?hasSV()?sv().chi2():0", float, doc="chi2 of SV fit", precision=8, lazyEval=True),
```

Comment:
```suggestion
    refitSVchi2 = Var("?hasSV()?sv().normalizedChi2():0", float, doc="reduced chi2, i.e. chi2/ndof, of SV fit", precision=8, lazyEval=True),
```
I think this is causing the difference in NANO.

Reply:
fixed. Thanks again!